json-schemas/Dataset.json

```diff
@@ -0,0 +1,325 @@
+{
+  "additionalProperties": false,
+  "description": "Model for dataset on the AMP-ALS portal",
+  "properties": {
+    "alternateName": {
+      "description": "An alternate name that can be used for search and discovery improvement.",
+      "title": "Alternate Name",
+      "type": "string"
+    },
+    "citation": {
+      "description": "identifies academic articles that are recommended by the data provider be cited in addition to the dataset itself.",
+      "items": {
+        "type": "string"
+      },
+      "title": "Citation",
+      "type": "array"
+    },
+    "collection": {
+      "description": "Associated collection",
+      "title": "Collection",
+      "enum": [
+        "Answer ALS",
+        "CReATe Consortium",
+        "Genomic Translation for ALS Care (GTAC)",
+        "New York Genome Center ALS Consortium",
+        "National Institutes of Health",
+        "Project MinE"
+      ],
+      "type": "string"
+    },
+    "contributor": {
+      "description": "Institution or person responsible for collecting and managing the files and records in the dataset.",
+      "items": {
+        "type": "string"
+      },
+      "title": "Contributor(s)",
+      "type": "array"
+    },
+    "creator": {
+      "description": "Properly formatted name of the organization or person that is creator of the dataset (e.g. \"NF-OSI\" or \"Robert Allaway\"), not an id. Most preferred is the project PI or data lead who's responsible for its conception and initial content creation.",
+      "items": {
+        "type": "string"
+      },
+      "title": "Creator",
+      "type": "array"
+    },
+    "datePublished": {
+      "description": "Date data were published/available on Synapse.",
+      "title": "Date Published",
+      "type": "integer"
+    },
+    "description": {
+      "description": "Blurb for the dataset; should be no more than 500 characters.",
+      "title": "Description",
+      "type": "string"
+    },
+    "funder": {
+      "items": {
+        "type": "string"
+      },
+      "title": "Funder",
+      "type": "array"
+    },
+    "includedInDataCatalog": {
+      "description": "Link(s) to known data catalog(s) the dataset is included in.",
+      "title": "Included In Data Catalog",
+      "type": "string"
+    },
+    "keywords": {
+      "items": {
+        "type": "string"
+      },
+      "title": "Keywords",
+      "type": "array"
+    },
+    "license": {
+      "description": "Unless information for license is clear, this should default to UNKNOWN.",
```

Comment:
```
      "description": "Unless information for license is clear, this should default to UNKNOWN.",
```

Would it be worth adding licences as SPDX identifiers? (probably just the creative commons subset, or other dataset relevant ones, e.g. not `zlib`)

Reply:
https://spdx.org/licenses/

Reply:
NVM I see the  license yaml below. Should we include those as enums here?

Reply:
Yeah, let's make a ticket for that for now.